### PR TITLE
Add HERE proprietary license

### PR DESCRIPTION
**Pull request by haikoschol**

### CODE REVIEW

**Comment by pombredanne** [comment]
@haikoschol thanks! I re-spun the Windows build that fails sometimes for no fathomable reasons

**Comment by sschuberth** [comment]
Sorry for a probably stupid question, but how "tolerant" is ScanCode to license texts defined like this? Will it find occurrences

- with a different copyright year,
- with different line wrapping,
- with leading programming-language-specific comment characters?

**Comment by pombredanne** [comment]
@sschuberth re
> how "tolerant" is ScanCode to license texts defined like this? Will it find occurrences
> - with a different copyright year,
> - with different line wrapping,
> - with leading programming-language-specific comment characters?

yes, yes and yes: tolerant to all these. But I would prefer to leave out the copyright notice from the text since this is detected by the copyright scanner separately.
And I can see a minor bug in the copyrights detection:
`Copyright (C) 2016-2018 HERE Global B.V. and its affiliate(s).` is detected as  `Copyright (C) 2016-2018 HERE Global B.V.` see #965

**Comment by pombredanne** [comment]
Looking good: thanks! I am pushing the copyright related fix For #965 separately.

File: src/licensedcode/data/licenses/here-proprietary.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+key: here-proprietary
+name: HERE Proprietary License
+short_name: HERE Proprietary
+category: Proprietary
+owner: HERE Global B.V.
```
Comment (sschuberth) [inline src/licensedcode/data/licenses/here-proprietary.yml]: @pombredanne Does this line need to be adjusted to contain "and its affiliate(s)" after #965 has been addressed?

**Pull request merged**